modules/social_features/social_content_report/src/Plugin/views/field/ReportContentType.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Drupal\social_content_report\Plugin\views\field;
+
+use Drupal\views\Plugin\views\field\FieldPluginBase;
+use Drupal\views\ResultRow;
+
+/**
+ * Field handler to present the entity type or bundle type for Nodes.
+ *
+ * @ingroup views_field_handlers
+ *
+ * @ViewsField("flagging_entity_bundle_type")
+ */
+class ReportContentType extends FieldPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function query() {}
+
+  /**
+   * {@inheritdoc}
+   */
+  public function render(ResultRow $row) {
+    /** @var \Drupal\flag\FlaggingInterface $flagging */
+    $flagging = $this
+      ->getEntity($row);
+
+    if ($flagging->getFlaggable()->getEntityTypeId() === 'node') {
```

Comment:
Why are we going the difficult route here?

```
$node->getEntityType()->getLabel()
```

Would return `TranslatableMarkup` with the translated label for the node type (e.g. `Content`).

Since `$flagging->getFlaggable()` is a node (given its `entity_type_id`) the line in the else statement would work here too:

```
$flagging->getFlaggable()->getEntityType()->getLabel();
```

Reply:
Seems legit. If it works, feel free to adjust it!